pymeasure/instruments/kepco/kepcobop.py

```diff
@@ -0,0 +1,180 @@
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+# List of Kepco BOP's (Vmax,Imax) [single channel output]
+# 100 W:
+# 5-20
+# 20-5
+# 50-2
+# 100-1
+
+# 200 W:
+# 5-30
+# 20-10
+# 36-6
+# 50-4
+# 72-3
+# 100-2
+# 200-1
+
+# 400 W:
+# 20-20
+# 36-12
+# 50-8
+# 72-6
+# 100-4
+
+from pymeasure.instruments import Instrument, SCPIMixin
+from pymeasure.instruments.validators import strict_discrete_set, \
+    truncated_range
+
+OPERATING_MODES = ['VOLT', 'CURR']
+
+
+class KepcoBOP3612(SCPIMixin, Instrument):
+    """
+    Represents the Kepco BOP 36-12 (M or D) 400 W bipolar power supply
+    fitted with BIT 4886 digital interface card (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+    """
+    _Vmax = 36
+    _Imax = 12
+
+    def __init__(self, adapter, name="Kepco BOP 36-12 Bipolar Power Supply",
+                 baud_rate=9600, read_termination="\n", write_termination="\n",
+                 **kwargs):
+        super().__init__(
+            adapter,
+            name,
+            baud_rate=baud_rate,
+            read_termination=read_termination,
+            write_termination=write_termination,
+            **kwargs
+        )
+
+    output_enabled = Instrument.control(
+        "OUTPut?",
+        "OUTPut %d",
+        """
+        Control whether the source is enabled, takes values True or False (bool)
+        """,
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True,
+    )
+
+    def beep(self):
+        """Causes the unit to emit a brief audible tone."""
+        self.write("SYSTem:BEEP")
+
+    confidence_test = Instrument.measurement(
+        "*TST?",
+        """
+        Power supply interface self-test procedure.
+        Returns 0 if all tests passed,
+        otherwise corresponding error code.
+        """,
+        cast=int
+    )
+
+    bop_test = Instrument.measurement(
+        "DIAG:TST?",
+        """
+        Power supply self-test (includes interface plus BOP operation).
+        Caution: Output will switch on and swing to maximum values.
+        Disconnect any load before testing.
+        Reutnrs 0 if all tests passed,
+        otherwise corresponding error code.
+        """,
+        cast=int
+    )
+
+    def wait_to_continue(self):
+        """ Causes the power supply to wait until all previously issued
+        commands and queries are complete before executing subsequent
+        commands or queries. """
+        self.write("*WAI")
+
+    voltage = Instrument.measurement(
+        "MEASure:VOLTage?",
+        """
+        Measures actual voltage that is across the output terminals.
```

Comment:
As explained above, it should start with `"Measure"` or `"Get"`:
```suggestion
        Measure current voltage across the output terminals in Volts.
```

The units are good to mention.